com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -554,16 +552,13 @@ static void OnAdditionalProbesBakeCompleted()
             // Reset index
             UnityEditor.Experimental.Lightmapping.SetAdditionalBakedProbes(m_BakingBatch.index, null);
 
-            // Map from each scene to an existing reference volume
+            // Map from each scene to an existing reference volume, and each reference volume to a new asset
             var scene2Data = new Dictionary<Scene, ProbeVolumePerSceneData>();
-            foreach (var data in GameObject.FindObjectsOfType<ProbeVolumePerSceneData>())
-                scene2Data[data.gameObject.scene] = data;
-
-            // Map from each reference volume to its asset
             var data2Asset = new Dictionary<ProbeVolumePerSceneData, ProbeVolumeAsset>();
-            foreach (var data in scene2Data.Values)
+            foreach (var data in GameObject.FindObjectsOfType<ProbeVolumePerSceneData>())
             {
-                data2Asset[data] = ProbeVolumeAsset.CreateAsset(data.gameObject.scene);
+                scene2Data[data.gameObject.scene] = data;
+                data2Asset[data] = data.CreateAssetForCurrentState();
```

Comment:
This also has other side effects, why changing from what was there before? Responsability of creating the asset should not be on the data itself.

Reply:
i thought it would make sense that the data does it since it manages the states, but i don't really mind, i reverted it to how it was before